content/v2.0/reference/key-concepts.md

```diff
@@ -0,0 +1,184 @@
+
+##### Tag keys
+
+The tag keys in the sample data are `location` and `scientist`.
+
+##### Tag values
+
+The tag key `location` has two tag values: `1` and `2`.
+The tag key `scientist` also has two tag values: `anderson` and `mullen`.
+
+##### Tag sets
+
+The collection of tag key-value pairs make up a tag set. The sample data includes the following four tag sets:
+
+- `location = 1`, `scientist = anderson`
+- `location = 2`, `scientist = anderson`
+- `location = 1`, `scientist = mullen`
+- `location = 2`, `scientist = mullen`
+
+#### Tags are indexed
+
+Tags are optional. You don't need tags in your data structure, but it's typically a good idea to include tags.
+Because tags are indexed, queries on tags are faster than queries on fields. This makes tags ideal for storing commonly-queried metadata.
+
+#### Why your schema matters
+
+If most of your queries focus on values in the fields, for example, a query to find when 23 bees were counted:
+
+`SELECT * FROM census WHERE bees = 23`
+
+InfluxDB scans every field value in the dataset for `bees` before the query returns a response. If our sample `census` data grew to millions of rows, to optimize your query, you could rearrange your [schema](/v2.0/reference/glossary/#schema) so the fields (`bees` and `ants`) becomes tags and the tags (`location` and `scientist`) become fields:
+
+| _time                 | _measurement | _field | _value |<span class="tooltip" data-tooltip-text="Tag key">bees</span>|<span class="tooltip" data-tooltip-text="Tag key">ants</span>|
+|:----------------------|--------------|--------|--------|-------|-------|
+| 2019-08-18T00:00:00Z | census |scientist |anderson   |   23 | |
+| <span class="tooltip" data-tooltip-text="Timestamp">2015-08-18T00:00:00Z</span> | <span class="tooltip" data-tooltip-text="Measurement name">census</span> |<span class="tooltip" data-tooltip-text="Field key">scientist</span> | <span class="tooltip" data-tooltip-text="Field value">mullen</span> | <span class="tooltip" data-tooltip-text="Tag value">30</span>  |
+| 2019-08-18T00:06:00Z | census |scientist| anderson| 28 |    |
+|                      |        |         |         |    |    |
+| 2019-08-18T00:00:00Z | census |location | 1       | 23 |    |
+| 2019-08-18T00:00:00Z | census |location | 1       | 30 |    |
+| 2019-08-18T00:06:00Z | census |location | 2       | 28 |    |
+|                      |        |         |         |    |    |
+| 2019-08-18T00:06:00Z | census |location | 2       |    | 3  |
+
+Now that `bees` and `ants` are tags, InfluxDB doesn't have to scan all `_field` and `_value` columns. This makes your queries faster.
+
+#### Series
+
+Now that you're familiar with measurements, field sets, and tag sets, it's time to discuss **series keys** and **series**. A series key is the collection of data that shares a measurement, tag set, and field key. For example, the [sample data](#sample-data) includes four unique series:
```

Comment:
...it's time to discuss **series** and **series keys**. A series is a the collection of points that share a measurement, tag set, and field key. For example, the [sample data](#sample-data) includes two unique series:

Reply:
@sanderson Do we still have 4 unique series with "location = 1,scientist=anderson bees" and "location = 2,scientist=anderson bees" ?